docs/development/core/public/kibana-plugin-core-public.appnavoptions.euiicontype.md

```diff
@@ -4,7 +4,7 @@
 
 ## AppNavOptions.euiIconType property
 
-A EUI iconType that will be used for the app's icon. This icon takes precendence over the `icon` property.
```

Comment:
All of the changes in the `/docs/development` directory are autogenerated from code comments. So in order for these updates to not be overwritten, you'll need to update them in the code itself and then run `yarn docs:acceptApiChanges`.

For example, this particular typo originates from `/src/core/public/application/types.ts`